examples/basic/axes/twin_axes.py

```diff
@@ -11,23 +11,26 @@
 
 from bokeh.models import LinearAxis, Range1d
 from bokeh.plotting import figure, show
+from bokeh.palettes import Sunset6
 
 x = arange(-2*pi, 2*pi, 0.2)
 y = sin(x)
 y2 = linspace(0, 100, len(x))
 
+blueshade, redshade = Sunset6[2], Sunset6[5]
```

Comment:
To be consistent with the rest of the codebase, we'd use "snake case" here, e.g. `blue_shade` but I think it's fine to just use "blue" and "red"
```suggestion
blue, red = Sunset6[2], Sunset6[5]
```